docs/nlu/components.rst

```diff
@@ -256,23 +256,37 @@ CountVectorsFeaturizer
 Intent Classifiers
 ------------------
 
+
+.. _keyword_intent_classifier:
+
 KeywordIntentClassifier
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-:Short: Simple keyword matching intent classifier. Not intended to be used.
+:Short: Simple keyword matching intent classifier.
```

Comment:
I'd maybe still leave a note saying something along the lines of that it's not sensible to use longterm

Reply:
👍